### ISSUE TITLE
Create a script to get current gas fees on ethereum

### ISSUE THREAD

**Issue by mbcse** [theblockchainchief/web3-hub#24]
Create a script to get gas fees from all possible public apis or resources. Basically the script should predict the best possible gas fees so that transaction does not fail. For example few sources to get base can be web3js function, gas station, etc. You need to aggregate result and give best possible value.

Can code in any language python, javascript, rust, etc

**Comment by legendarykamal** [comment]
hi @DarkFalc0n  are you still working on this , If not can i get assigned @mbcse ?

**Comment by DarkFalc0n** [comment]
> hi @DarkFalc0n  are you still working on this , If not can i get assigned @mbcse ?

No, not working on it currently, you can go on to start working on it

**Comment by mohammednudman** [comment]
@mbcse Can you assign me this work

**Comment by fykaa** [comment]
@legendarykamal I am assigning you this issue. Looking forward to your contribution to the same. Have a great time!

**Comment by fykaa** [comment]
@legendarykamal any updates?
